crates/chia-consensus/src/gen/validation_error.rs

```diff
@@ -95,39 +204,145 @@ pub fn first(a: &Allocator, n: NodePtr) -> Result<NodePtr, ValidationErr> {
 impl From<ErrorCode> for u32 {
     fn from(err: ErrorCode) -> u32 {
         match err {
```

Comment:
if we make this error code type hold the union of all python and rust errors, maybe we don't even need this conversion function. Or maybe we can just relying on the error value just being the rank of the error code list.